src/components/Personalization/constants/eventType.js

```diff
@@ -12,3 +12,7 @@ governing permissions and limitations under the License.
 
 export const DISPLAY = "decisioning.propositionDisplay";
 export const INTERACT = "decisioning.propositionInteract";
+export const EventType = {
```

Comment:
This is confusing. Perhaps call this proposition event type? Maybe in a separate file.